Zend/zend_enum.c

```diff
@@ -36,6 +36,30 @@ ZEND_API zend_class_entry *zend_ce_unit_enum;
 ZEND_API zend_class_entry *zend_ce_backed_enum;
 ZEND_API zend_object_handlers zend_enum_object_handlers;
 
+// We are going to need some argument info for ::from() and ::tryFrom() for
```

Comment:
Nit: Please stick with the C-style comments we're generally using `/* */` with ` *` for the non-start & end line.